eng/common/scripts/Package-Properties.ps1

```diff
@@ -395,7 +395,7 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
                     $includedForValidation = ($filePath -like (Join-Path "$resolvedRelativePath" "*") -or $filePath -eq $resolvedRelativePath)
                     $shouldInclude = $shouldInclude -or $includedForValidation
                     if ($includedForValidation) {
-                        $pkg.IncludedForValidation = $true
+                        $pkg.IncludedForValidation = $false
```

Comment:
Do we need to explicitly set this to false or is that the default and we can just skip this if block completely?

Reply:
We want to keep the `break` in place as there is no reason to continue iterating over files for this package, but I do agree about removing the `$pkg.IncludedForValidation` setting.